scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -261,7 +262,12 @@ def generate_command(self):
         handler: Handler = self.instance.handler
 
         if handler.options.verbose > 1:
-            command.append('--log-level=DEBUG')
+            command.extend([
+                '--log-level=DEBUG',
+                '-o', 'log_cli=true',
+                '--log-cli-format=%(levelname)s %(message)s',
+                f'--log-file={self.pytest_log_file_path}'
+            ])
 
```

Comment:
I wondering if this is good idea. I think that Twister is run "without verbosity" mainly in CI, where saved outputs/logs should be limited somehow, to avoid situation when thousands tests generate additionally megabytes of output, by saving this log files. If some test fails, the most often scenario is that you try to run it once again locally with full verbosity (`-vv`) and then you will have full access to `twister_harness.log` file.

Reply:
If you compare size of these logs (just a text) with another artefacts from building, they are rather small.
It is good to have such a debug log available, especially if test fails. It is not always easy to reproduce failing scenario (test may be unstable, and found error only during CI execution, or you do not have a device to run such a test).
In many cases, when you find failing log, then you know, what is wrong (without need of setup it locally).
And, on the other hand, in CI you can have removed all unwanted artefacts after execution tests... just keep only logs from failed tests (and than every input is welcome)

Reply:
Ok, changed as you proposed.